src/authentication/CredentialsExtractor.ts

```diff
@@ -4,6 +4,5 @@ import type { Credentials } from './Credentials';
 
 /**
  * Responsible for extracting credentials from an incoming request.
- * Will return `null` if no credentials were found.
```

Comment:
@joachimvh This seem wrong: rather, it returns `{ webID: undefined }`. Is that correct?

Reply:
Yea that's probably just an outdated comment